databases/cockroachdb/README.md

```diff
@@ -0,0 +1,215 @@
+# CockroachDB example
+
+This example shows how to connect Prisma to a CockroachDB database, create the database schema with raw SQL, use introspection to populate the Prisma schema and use [Prisma Client](https://www.prisma.io/client) in a TypeScript script to read and write data in the database.
+
+> **Note:** CockroachDB support in Prisma is currently in **Preview** and doesn't support Prisma Migrate. For this reason, the database schema is created with raw SQL.
+
+The example consists of two parts:
+
+- `tests/prisma.test.ts`: Jest test (in TypeScript) with a variety of Prisma Client queries and assertions to showcase access patterns
+- `src/script.ts`: Node.js script with queries similar to the ones in the test.
+
+## Prerequisites
+
+- Node.js installed.
+- [CockroachDB installed](https://www.cockroachlabs.com/docs/v21.2/install-cockroachdb.html) so you can run CockroachDB locally and use the CockroachDB client (interactive shell) .
```

Comment:
```suggestion
- [CockroachDB installed](https://www.cockroachlabs.com/docs/stable/install-cockroachdb.html) so you can run CockroachDB locally and use the CockroachDB client (interactive shell) .
```
`stable` points to same page as v21.2 but will be updated automatically on next release, so probably better to use that